internal/distro/rhel7/images.go

```diff
@@ -141,6 +142,21 @@ func osCustomizations(
 		osc.FactAPIType = options.Facts.ApiType
 	}
 
+	var err error
+	osc.Directories, err = blueprint.DirectoryCustomizationsToFsNodeDirectories(c.GetDirectories())
```

Comment:
Can you generalize this chunk of code to someting like ?
```
osc.Directoriies, osc.Files, err = bluepring.somename(...)
```
To avoid copy pasting this 3 times.

Reply:
I admit that his is a bit verbose, but this is unfortunately the way how error handling works in Go. I do not consider this to be an issue to be honest.

I do not want to unnecessarily tie Files and Directories customization together when not needed. These are two different customizations. We do not do what you are suggesting for any other customizations (e.g. users / groups).

Lastly, the fact that this is copied three times in each distro is a result of how we define distros, not of the fact that there is a separate function for each customization, as there already is for those existing ones. Instead, we should aim for deduplicating `distro/<distro>/images.go` where it makes sense, because these are very similar. However this was not the goal of the rework that @achilleas-k did in all distros, but it is AFAIK something that we would like to eventually look at.